llvm/lib/Support/ARMBuildAttrs.cpp

```diff
@@ -67,7 +69,56 @@ static const TagNameItem tagData[] = {
     {ARMBuildAttrs::ABI_align_preserved, "Tag_ABI_align8_preserved"},
 };
 
-constexpr TagNameMap ARMAttributeTags{tagData};
-const TagNameMap &llvm::ARMBuildAttrs::getARMAttributeTags() {
-  return ARMAttributeTags;
+const TagNameItem AVAttributeTags[] = {
```

Comment:
These are downstream changes we have to work with armlink, and shouldn't be included here.